fluent/handler.py

```diff
@@ -51,14 +51,14 @@ def format(self, record):
         data = dict([(key, value % record.__dict__)
                      for key, value in self._fmt_dict.items()])
 
-        self._structuring(data, record.msg)
+        self._structuring(data, record.msg, record.args)
```

Comment:
Since `record.msg` can be a dict, that does not work, but by passing the record to `_structuring` as you did in #43 is indeed better.